tests/test_models.py

```diff
@@ -2541,7 +2536,7 @@ class TestEnterpriseCustomerInviteKey(unittest.TestCase):
             'expiration_date': EXPIRED_DATE,
             'usage_limit': 1,
             'usage_count': 1,
-            'is_active': True,
+            'is_active': False,
```

Comment:
why did this test input have to change?

Reply:
Good catch, it doesn't need to change, since the `EXPIRED_DATE` guarantees that `expected_is_valid` will be false.